examples/background_task.py

```diff
@@ -13,18 +14,19 @@ def __init__(self, *args, **kwargs):
         self.my_background_task.start()
 
     async def on_ready(self):
-        print(f'Logged in as {self.user} (ID: {self.user.id})')
-        print('------')
+        print(f"Logged in as {self.user} (ID: {self.user.id})")
+        print("------")
 
-    @tasks.loop(seconds=60) # task runs every 60 seconds
+    @tasks.loop(seconds=60)  # task runs every 60 seconds
     async def my_background_task(self):
-        channel = self.get_channel(1234567) # channel ID goes here
+        channel = self.get_channel(1234567)  # channel ID goes here
         self.counter += 1
         await channel.send(self.counter)
 
     @my_background_task.before_loop
     async def before_my_task(self):
-        await self.wait_until_ready() # wait until the bot logs in
+        await self.wait_until_ready()  # wait until the bot logs in
+
 
 client = MyClient()
-client.run('token')
+client.run("token")
```

Comment:
The changes here don't appear to have anything to do with slash commands (same with many examples).

Nice to keep PRs on task.